sycl/source/detail/event_impl.cpp

```diff
@@ -342,7 +342,18 @@ void HostProfilingInfo::start() { StartTime = getTimestamp(); }
 void HostProfilingInfo::end() { EndTime = getTimestamp(); }
 
 pi_native_handle event_impl::getNative() const {
+  if (!MContext) {
+    static context SyclContext;
```

Comment:
Is it possible to avoid static variable? What's its life-time? When is this context destroyed?
My best guess is that it's better to employ global handler object to encapsulate this context.